src/mt_protocol.cpp

```diff
@@ -212,6 +212,12 @@ bool handle_packet(uint32_t now, size_t payload_len) {
   memmove(pb_buf, pb_buf+4+payload_len, PB_BUFSIZE-4-payload_len);
   pb_size -= 4 + payload_len;
 
+  // Be prepared to request a node report to re-establish flow after an MT reboot
+  meshtastic_ToRadio toRadio = meshtastic_ToRadio_init_default;
+  toRadio.which_payload_variant = meshtastic_ToRadio_want_config_id_tag;
+  want_config_id = random(0x7FffFFff);  // random() can't handle anything bigger
+  toRadio.want_config_id = want_config_id;
+
```

Comment:
do we want the config again? if not, use the magic 'skip config'

Reply:
Is it possible the node rebooted due to a config change?  Maybe getting the config again could be useful?

I'm looking around but cant find a reference to `skip config`

Reply:
I think caveman referred to the [`SPECIAL_NONCE`](https://github.com/meshtastic/firmware/blob/92225eb6c3ac60a12995432a4bfac0ee3641f093/src/mesh/PhoneAPI.h#L21). It will only skip sending NodeInfos of other nodes.

Reply:
That makes sense to only pull in your own config and not other nodes.  I dug around and have determined that implementing this is outside of my current skillset.

Reply:
Replacing the random `want_config_id` with 69420 should do it 😀

Reply:
Ya know I'd typed that in and thought, "there's no way it's that easy" and gave up.  Also gave my wife a laugh with that one.